docs/index.md

```diff
@@ -33,7 +33,7 @@ A list of the hosting we have, who provides it, and what capacity it has
 
 ### Rackspace
   * Previously sponsored, [being phased out](https://community.theforeman.org/t/rfc-moving-off-of-rackspace-infrastructure/17932)
-  * Hosts Jenkins and Foreman
+  * Used for Mailgun (to send email from Redmine and Discourse) and CloudFiles (to store Fastly CDN logs)
```

Comment:
The later is not correct anymore, @ares deleted the logs

Reply:
the first one: yes

the second: why?

Reply:
I may be tired and somehow remembered web02, but that was web02.rackspace.